charts/posthog/tests/clickhouse-instance.yaml

```diff
@@ -305,3 +338,21 @@ tests:
     asserts:
       - matchSnapshot:
           path: spec.configuration.clusters[0].layout
+
+  - it: podAnnotations should not be set by default
+    asserts:
+      - equal:
+          path: spec.templates.podTemplates[0].metadata
```

Comment:
Should this be spec.templates.podTemplates[0].metadata.annotations ? Or we should update the test name to be explicitly that metadata shouldn't be set.